priv/repo/migrations/20210518085818_add_started_completed_to_workflow_runs.exs

```diff
@@ -0,0 +1,10 @@
+defmodule Dashy.Repo.Migrations.AddStartedCompletedToWorkflowRuns do
+  use Ecto.Migration
+
+  def change do
+    alter table("workflow_runs") do
+      add :started_at, :utc_datetime, null: true
+      add :completed_at, :utc_datetime, null: true
```

Comment:
```suggestion
      add :started_at, :utc_datetime_usec, null: true
      add :completed_at, :utc_datetime_usec, null: true
```

This way it saves microseconds, otherwise it rounds everything to seconds.

Reply:
Mmm, but it's not used in the other tables, so I think it would be the same, right? We can open a new PR changing it everywhere if the GA API gives you that info.

Reply:
Ok!